notebooks/resource_allocation/rec_mem_req_lim.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Same comment as on the CPU notebook :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/operate-first-jupyterhub-analysis/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>